src/components/UsersSuggestionDropdown.js

```diff
@@ -1,8 +1,9 @@
 import React, { useEffect, useRef, useState } from 'react'
 import axios from 'axios'
 import { trackPromise, usePromiseTracker } from 'react-promise-tracker'
-import { apiURL } from '../config.json'
+import * as config from '../config.json'
 import LoadingIndicator from '../modules/LoadingIndicator'
+const { apiURL } = config
```

Comment:
Why is it updated? You are basically writing 2 lines which were already done in 1 line.

Reply:
It was returning a webpack-based error and creating issue in tests, so I had to replace it.

Reply:
what was the error?

Reply:
> Member



> what was the error? The path should be to fix the error, not break something which was done in the correct way.

![image](https://user-images.githubusercontent.com/75155230/151619823-7e05b2db-2592-41b4-a4d3-8179ad6e658c.png)

Reply:
@Abhirup-99 This was the error

Reply:
Didn't realise it was a JSON. Ignore comments.